test/e2e/app-launch.e2e.test.js

```diff
@@ -77,21 +77,6 @@ test.describe.serial('Application launch', async () => {
     expect(new URL(await mainWindow.url()).pathname).toBe('/dashboard')
   })
 
-  test('wait for Saturn node to get ready', async () => {
```

Comment:
I removed this for simplicity, but am not very sure about it. Why did we have this `isReady` check? Is it for e2e tests to verify that saturn can boot? In this case, that should already be covered by `core`'s testsuite

Reply:
Ok, good point about this being different now with Node.js. Let's add something like it back (hopefully now simpler) when it becomes a problem.